File: cash.c
// Include libraries
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollar;
    do
    {
        // Prompt the user to enter the amount of change they owe
        dollar = get_float("Change owed: ");
    }
    // This occurs is dollars is less than or equal to 0
    while (dollar <= 0);
    
    int cents = round(dollar*100);
    int coins = 0;
    
    // Subtract the number of cents if the user enters 25 cents or greater
    while (cents >= 25)
    {
        cents -= 25; 
        coins++;
    }
    
    // Subtract the number of cents if the user enters 10 cents or greater
    while (cents >= 10)
    {
        cents -= 10;
        coins++;
    }
    
    // Subtract the number of cents if the user enters 5 cents or greater
    while (cents >= 5)
    {
        cents -= 5;
        coins++;
    }
    
    // Subtract the number of cents if the user enters 1 cent or greater
    while (cents >= 1)
    {
        cents -= 1;
        coins++;
    }
    
    // Tell the user how much coins they need to pay
    printf("You will need at least %i coins", coins);
}

File: cash.py
# Prompt the user to change
from cs50 import get_int
while True:
    cents = get_int("Change owed: ")
    if (cents > 0):
        break
cents = round(cents * 100)
track = 0
# The number of quarters
while cents >= 25:
    cents = cents - 25
    track += 1
# The number of dimes
while cents >= 10:
    cents = cents - 10
    track += 1
# The number of nickels
while cents >= 5:
    cents = cents - 5
    track += 1
# The number of pennies
while cents >= 1:
    cents = cents - 1
    track += 1
# Print the total number of coins
print("The total amount of coins: ", track)